### DISCUSSION TITLE
Is there a way to detect/ignore realtime event sent from same client

### DISCUSSION THREAD

**Discussion by T04435** [supabase/supabase#31620] [Questions] [answered]
I have an app with a table with CRUD actions and realtime on it.

So I want to know if there is a way to ignore the realtime payload on the same client that triggered the event.

```
[client1] => insert => nothing
[client2] => gets above payload
```

Thanks.

**Comment by GaryAustin1** [comment, marked as answer]
I assume you are talking about Postgres changes, if so then no.  All clients receive the changes their RLS allows.
